src/tests/quantuminspire/qiskit/test_backend_qx.py

```diff
@@ -541,7 +596,7 @@ def run_histogram_test(self, single_experiment, mock_result1, mock_result2, expe
         self.mock_api.get_jobs_from_project.return_value = [jobs]
         job = QIJob('backend', '42', self.mock_api)
 
-        result = self.simulator.get_experiment_results(job)
+        result = self.simulator.get_experiment_results(job, False)
```

Comment:
The meaning of the second argument is not clear. Consider using the explicit parameter name: `only_latest_run=False`

Reply:
Done